src/DOMmodule.js

```diff
@@ -0,0 +1,497 @@
+/* eslint-disable max-len */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/no-cycle */
+import {
+  getListsFromStorage, AddItemToList, allTasks, createTodoItem, hideForm,
+} from './Todos';
+
+const DOMmodule = () => {
+  const mainContainer = document.createElement('div');
+  const headerContainer = document.createElement('div');
+  const contentContainer = document.createElement('div');
+  const todosContainer = document.createElement('div');
+  const rowContainer = document.createElement('div');
+  const messages = document.createElement('div');
+  const navContainer = document.createElement('div');
+  const titleNav = document.createElement('h3');
+  const buttonAddContainer = document.createElement('div');
+  const listContainer = document.createElement('div');
+  const textAdd = document.createElement('div');
+  const nameListBox = document.createElement('input');
+  const plusBtn = document.createElement('button');
+  const plusBtnList = document.createElement('button');
+  const cancelBtn = document.createElement('button');
+  const titleContainer = document.createElement('div');
+  const todoContent = document.createElement('div');
+  const taskContainer = document.createElement('div');
+  const taskBelowContainer = document.createElement('div');
+  let addContainer = document.getElementById('btn-add-list');
+  let textHide = document.getElementById('add-text');
+  const ulTitleContainer = document.createElement('ul');
+  const titleDone = document.createElement('li');
+  const titleName = document.createElement('li');
+  const titleDescription = document.createElement('li');
+  const titleDate = document.createElement('li');
+  const titleAction = document.createElement('li');
+  const EditTask = document.createElement('div');
+  const AddTask = document.createElement('div');
+  const CancelTask = document.createElement('div');
+  const itemListName = document.createElement('div');
+
+  const renderMessages = (text, color) => {
+    const message = document.getElementById('messages');
+    message.innerText = text;
+    message.style.backgroundColor = color;
+    message.classList.remove('d-none');
+
+    const delayInMilliseconds = 5000;
+
+    setTimeout(() => {
+      message.innerText = '';
+      message.className = 'd-none';
+    }, delayInMilliseconds);
+  };
+
+
+  const renderSideBar = (() => {
+    navContainer.id = 'nav-container';
+    titleNav.textContent = 'Lists';
+    buttonAddContainer.id = 'btn-add-list';
+    titleNav.id = 'nav-title';
+    listContainer.id = 'list-container';
+    textAdd.innerText = '+ Add List';
+    textAdd.id = 'add-text';
+
+    mainContainer.appendChild(navContainer);
+    buttonAddContainer.appendChild(textAdd);
+    navContainer.appendChild(titleNav);
+    navContainer.appendChild(buttonAddContainer);
+    navContainer.appendChild(listContainer);
+    rowContainer.appendChild(navContainer);
+    // textAdd.onclick = showMenuAdd;
```

Comment:
kindly remove this comment.

Reply:
Sure i will do that before merge !!, thanks!